### PR TITLE
Give the customer the option to log the output of the healthcheck.

### PR DESCRIPTION
**Pull request by jeroenmaelbrancke**

### CODE REVIEW

**Comment by jeroenmaelbrancke** [comment]
Testing, Creating PR (1h)

File: ovs/log/healthcheck_logHandler.py [hunk 1]
```diff
@@ -123,11 +124,17 @@ def logger(self, message, module, log_type, unattended_mode_name, unattended_pri
         """
 
         try:
-            target = open('{0}/{1}'.format(self.HEALTHCHECK_DIR, self.HEALTHCHECK_FILE), 'a')
+            if self.enable:
+                target = open('{0}/{1}'.format(self.HEALTHCHECK_DIR, self.HEALTHCHECK_FILE), 'a')
+            else:
+                target = ''
+
             now = datetime.datetime.now()
```
Comment (kinvaris) [inline ovs/log/healthcheck_logHandler.py]: now can be added to the if statement above.

File: ovs/log/healthcheck_logHandler.py [hunk 2]
```diff
@@ -123,11 +124,17 @@ def logger(self, message, module, log_type, unattended_mode_name, unattended_pri
         """
 
         try:
-            target = open('{0}/{1}'.format(self.HEALTHCHECK_DIR, self.HEALTHCHECK_FILE), 'a')
+            if self.enable:
+                target = open('{0}/{1}'.format(self.HEALTHCHECK_DIR, self.HEALTHCHECK_FILE), 'a')
+            else:
+                target = ''
```
Comment (kinvaris) [inline ovs/log/healthcheck_logHandler.py]: target must be declared in the local variables above.

File: ovs/log/healthcheck_logHandler.py (hunk 1 above)
Comment (kinvaris) [inline ovs/log/healthcheck_logHandler.py]: `now` must be added to the if statement above

**Pull request merged**